open-build-service/notqmail.spec.in

```diff
@@ -0,0 +1,365 @@
+%sysusers_requires
+%endif
+%if 0%{?suse_version} >= 1500 || 0%{?sles_version} >= 15
+%sysusers_requires
+%endif
+
+%if %build_on_obs == 1
+BuildRoot: %{_tmppath}/%{name}-%{version}-build
+%endif
+
+%description
+notqmail is a community-driven fork of qmail, beginning where netqmail
+left off: providing stable, compatible, small releases to which existing
+qmail users can safely update.
+notqmail also aims higher: developing an extensible, easily packaged, and
+increasingly useful modern mail server.
+
+%package doc
+Summary: Documentations for the %{name} package
+%if %{undefined suse_version} && %{undefined sles_version}
+Group: System Environment/Base
+%else
+Group: Productivity/Networking/Email/Servers
+%endif
+Requires: notqmail
+BuildArch: noarch
+
+%description doc
+notqmail is a community-driven fork of qmail, beginning where netqmail
+left off: providing stable, compatible, small releases to which existing
+qmail users can safely update.
+notqmail also aims higher: developing an extensible, easily packaged, and
+increasingly useful modern mail server.
+This package contains the documentation for %{name}
+
+%{see_base}
+
+%prep
+%setup
+
+%build
+%if %{undefined fedora_version} && %{undefined centos_version} && %{undefined rhel_version} && %{undefined sles_version} && %{undefined suse_version} && %{undefined scientificlinux_version} && %{undefined openeuler} && %{undefined almalinux_version} && %{undefined rocky_version}
+  %sysusers_generate_pre %{S:4} notqmail
+%endif
+%if 0%{?suse_version} >= 1500 || 0%{?sles_version} >= 15
+  %sysusers_generate_pre %{S:4} notqmail
+%endif
+conf_cc=`head -1 conf-cc`
+echo "$conf_cc -g -fPIC" > conf-cc
+echo "cc -fPIE -pie"     > conf-ld
+echo %{qmaildir}         > conf-qmail
+make -s it man NROFF=true
+
+%install
+env DESTDIR=%{buildroot} ./instpackage
+%{__rm} -rf %{buildroot}%{qmaildir}/man/man3 %{buildroot}%{qmaildir}/man/cat* %{buildroot}%{qmaildir}/queue
+%{__mkdir_p} %{buildroot}%{_mandir}
+%{__mkdir_p} %{buildroot}%{nq_docdir}
+%{__mkdir_p} %{buildroot}%{qmaildir}/libexec
+install -m 700 instchown %{buildroot}%{qmaildir}/libexec
+install -m 700 instpackage %{buildroot}%{qmaildir}/libexec
+mv %{buildroot}%{qmaildir}/man/man? %{buildroot}%{_mandir}
+mv %{buildroot}%{qmaildir}/doc/* %{buildroot}%{nq_docdir}
+%{__cp} COPYRIGHT %{buildroot}%{nq_docdir}
+rmdir --ignore-fail-on-non-empty %{buildroot}%{qmaildir}/doc
+rmdir --ignore-fail-on-non-empty %{buildroot}%{qmaildir}/man
+%if 0%{?suse_version} >= 1120
+  %{__mkdir_p} %{buildroot}%{_sysconfdir}/permissions.d/
+  install -m 644 %{S:1} %{buildroot}%{_sysconfdir}/permissions.d/%{name}-permissions
+  install -m 644 %{S:2} %{buildroot}%{_sysconfdir}/permissions.d/%{name}-permissions.secure
```

Comment:
In reality none of the files are actually needed except for the SLES systems. But without qmail-queue being setuid, we cannot queue any mail without making insecure permissions for the queue directory. I have it there because generally I see distributions including all three (easy, secure and paranoid). These files are bit like the debian dpkg-override. I'm in two minds about removing the paranoid file. Having the file reminds me that there are three security settings for permissions. But yes, it is uselessly sitting out there.

Reply:
This has been addressed as part of commit #[679054c](https://github.com/notqmail/notqmail/pull/76/commits/679054c6130f8c5ee012714270234bf55cc921d7)